src/System.Net.Http/tests/FunctionalTests/XUnitAssemblyAttributes.cs

```diff
@@ -0,0 +1,8 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+// [ActiveIssue(35002)]: Disabling parallel execution of System.Net.Http.FunctionalTests
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true, MaxParallelThreads = 1)]
```

Comment:
If "DisableTestParallelization = true" then does it matter whether or not you change "MaxParallelThread" at all, or even have it mentioned in this file?

Reply:
I did some search online, having both is useful in some scenarios, for example: when some components in the test project are not compatible with the TPL, and xUnit uses TPL extensively, even when tests aren't being run in parallel.

I think it's fine to only keep `DisableTestParallelization = true` here.